config/default/forms/app/pregnancy.xml

```diff
@@ -967,7 +967,7 @@
       <bind nodeset="/pregnancy/inputs" relevant="./source = 'user'"/>
       <bind nodeset="/pregnancy/inputs/source" type="string"/>
       <bind nodeset="/pregnancy/inputs/source_id" type="string"/>
-      <bind nodeset="/pregnancy/inputs/user/contact_id" type="db:person"/>
+      <bind nodeset="/pregnancy/inputs/user/contact_id" type="select-contact"/>
```

Comment:
I think this has to be changed to "string", right?
Otherwise it's going to search for `select-contact` as the contact type.

Reply:
Done. Sorry for the confusion!